_content/tour/ind/maps.article

```diff
@@ -28,31 +28,30 @@ block of memory underneath.
 .play maps/example6.go
 .play maps/example7.go
 
-** Declaring And Constructing Maps
+** Mendeklarasikan dan Membuat Maps
 
-Declaring and constructing a map can be done in several ways.
+Mendeklarasikan dan membuat map dapat dilakukan dengan beberapa cara.
 
     type user struct {
         name     string
         username string
     }
 
-    // Construct a map set to its zero value,
-    // that can store user values based on a key of type string.
-    // Trying to use this map will result in a runtime error (panic).
+    // Buatlah map yang disetel ke nilai nol nya,
```

Comment:
instead of `disetel`, better to use `diatur`

Reply:
ack